### ISSUE TITLE
Can't discover sensor via mDNS 

### ISSUE THREAD

**Issue by nerdworldDE** [ebaauw/fp2-proxy#3]
For some reason i can't discover any devices:

```
fp2-proxy.js: /home/pi/.fp2-proxy: read
fp2-proxy discover: mDNS: searching...
fp2-proxy discover: mDNS: search done
{}
```

Also no config file is created in the home directory yet.
Is there a way to add a sensor manually via IP / MAC / ID?

**Comment by ebaauw** [comment]
No, the sensors cannot be added manually.  Make sure the sensors work in the Aqara app, and are visible to HomeKit.  The sensors need to be on the same subnet as the server running `fp2-proxy`.

Note that `fp2-proxy` is still work in progress.  The current version should connect to the sensors and to deCONZ, but doesn’t yet create the virtual sensors.

**Comment by JacekKac** [comment]
ho @ebaauw great work! still trying to discover my fp2 using your fp2-proxy. Steps i did: 
1/ Added to aquara home app which force me to add it also to me IOS home app. Did it. 
2/ Remove the sensor from homekit app to free the pairing limit. 
3/ fp2 is visible
`
pi@raspberrypi:~/fp2-proxy $ avahi-browse -a
+  wlan0 IPv4 Presence-Sensor-FP2-272C                      _aqara-setup._tcp    local
+  wlan0 IPv4 Presence-Sensor-FP2-272C                      _hap._tcp            local
+  wlan0 IPv4 Aqara-FP2-272C                                _Aqara-FP2._tcp      local
`

`
pi@raspberrypi:~/fp2-proxy $ node cli/fp2-proxy discover
{}
`

**Comment by JacekKac** [comment]
When i remove it from homekit 

`pi@raspberrypi:~/fp2-proxy $ avahi-browse -r _hap._tcp
+  wlan0 IPv4 Presence-Sensor-FP2-272C                      _hap._tcp            local
+  wlan0 IPv4 openHAB                                       _hap._tcp            local
=  wlan0 IPv4 Presence-Sensor-FP2-272C                      _hap._tcp            local
   hostname = [Presence-Sensor-FP2-272C.local]
   address = [192.168.0.213]
   port = [60222]
   txt = ["c#=1" "ff=2" "id=A2:88:ED:00:71:D1" "md=PS-S02E" "pv=1.1" "s#=2" "sf=0" "ci=10" "sh=JlOq9Q=="]
=  wlan0 IPv4 openHAB                                       _hap._tcp            local
   hostname = [192-168-0-136.local]
   address = [192.168.0.136]
   port = [9123]
   txt = ["id=8a:e0:1f:83:2b:19" "md=openHAB" "pv=1.1" "ci=2" "sh=79UOzQ==" "sf=0" "ff=0" "s#=1" "c#=387"]
^CGot SIGINT, quitting.
pi@raspberrypi:~/fp2-proxy $ avahi-browse -r _hap._tcp
+  wlan0 IPv4 Presence-Sensor-FP2-272C                      _hap._tcp            local
+  wlan0 IPv4 openHAB                                       _hap._tcp            local
=  wlan0 IPv4 Presence-Sensor-FP2-272C                      _hap._tcp            local
   hostname = [Presence-Sensor-FP2-272C.local]
   address = [192.168.0.213]
   port = [60222]
   txt = ["c#=1" "ff=2" "id=A2:88:ED:00:71:D1" "md=PS-S02E" "pv=1.1" "s#=2" "sf=1" "ci=10" "sh=JlOq9Q=="]
=  wlan0 IPv4 openHAB                                       _hap._tcp            local
   hostname = [192-168-0-136.local]
   address = [192.168.0.136]
   port = [9123]
   txt = ["id=8a:e0:1f:83:2b:19" "md=openHAB" "pv=1.1" "ci=2" "sh=79UOzQ==" "sf=0" "ff=0" "s#=1" "c#=387"]
^CGot SIGINT, quitting.
pi@raspberrypi:~/fp2-proxy $ node cli/fp2-proxy discover
{}
pi@raspberrypi:~/fp2-proxy $
`

**Comment by JacekKac** [comment]
Tried to pair with ID nad setup code also 

pi@raspberrypi:~/fp2-proxy $ avahi-browse -r _hap._tcp
+  wlan0 IPv4 Presence-Sensor-FP2-272C                      _hap._tcp            local
=  wlan0 IPv4 Presence-Sensor-FP2-272C                      _hap._tcp            local
   hostname = [Presence-Sensor-FP2-272C.local]
   address = [192.168.0.213]
   port = [60222]
   txt = ["c#=1" "ff=2" "id=A2:88:ED:00:71:D1" "md=PS-S02E" "pv=1.1" "s#=2" "sf=1" "ci=10" "sh=JlOq9Q=="]




pi@raspberrypi:~/fp2-proxy $ node cli/fp2-proxy pair A2:88:ED:00:71:D1 04668045
fp2-proxy pair: error: cannot find A2:88:ED:00:71:D1
pi@raspberrypi:~/fp2-proxy $ ping 192.168.0.213
PING 192.168.0.213 (192.168.0.213) 56(84) bytes of data.
64 bytes from 192.168.0.213: icmp_seq=1 ttl=255 time=186 ms
pi@raspberrypi:~/fp2-proxy $ node cli/fp2-proxy pair A2:88:ED:00:71:D1 04668045
fp2-proxy pair: error: cannot find A2:88:ED:00:71:D1
pi@raspberrypi:~/fp2-proxy $ ping 192.168.0.213
PING 192.168.0.213 (192.168.0.213) 56(84) bytes of data.
64 bytes from 192.168.0.213: icmp_seq=1 ttl=255 time=92.4 ms

pi@raspberrypi:~/fp2-proxy $ avahi-browse -r _hap._tcp
+  wlan0 IPv4 Presence-Sensor-FP2-272C                      _hap._tcp            local
=  wlan0 IPv4 Presence-Sensor-FP2-272C                      _hap._tcp            local
   hostname = [Presence-Sensor-FP2-272C.local]
   address = [192.168.0.213]
   port = [60222]
   txt = ["c#=1" "ff=2" "id=A2:88:ED:00:71:D1" "md=PS-S02E" "pv=1.1" "s#=2" "sf=1" "ci=10" "sh=JlOq9Q=="]

**Comment by JacekKac** [comment]
it's not possible from the aquara app to add the sensor to wifi and skip the homekit add so i am stucked :(

**Comment by ebaauw** [comment]
Please try the following:
- Reset the sensor, pair it with the Aqara app and with HomeKit;
- Check with `fp2-proxy discover` that you see the sensor over HAP.  It should report `pairable` as false;
- Remove the accessory from HomeKit.
- Check with `fp2-proxy discover` that you still see the sensor over HAP.  It should now report `pairable` as true;
- Run `fp2-proxy pair `_id_ _setup_code_.  The _setup_code_ is the HomeKit setup code, I think in the form _XXX-XX-XXX_, but it could also be _XXXX-XXXX_.

Note that you need to pair the FP2 with `fp2-proxy` shortly after unpairing with HomeKit, or it will stop advertising.

Also note that `fp2-proxy` is still work in progress, and won't yet proxy the FP2 state to deCONZ.  But if you run `fp2-proxy -D` you should see debug messages when the FP2 state changes.
